src/components/selectionBoxContainer.ts

```diff
@@ -0,0 +1,28 @@
+///<reference path="../reference.ts" />
+
+module Plottable {
+export module Component {
+  export class SelectionBoxContainer extends AbstractComponent {
+    private _selectionBoxes: Entity.SelectionBox[] = [];
+
+    constructor() {
+      super();
+      this.classed("selection-box-container", true);
+    }
+
+    protected _setup() {
+      super._setup();
+      this._selectionBoxes.push(new Plottable.Entity.SelectionBox(this._content));
```

Comment:
Hmm, this._content seems to be protected...  Is using this._content appropriate?